### PR TITLE
CANS-541 added Ferb api to system information gem

### PR DESCRIPTION
**Pull request by AVandana-OSI**
https://osi-cwds.atlassian.net/browse/CANS-541

**Description**: Added Ferb API health check

### CODE REVIEW

File: spec/system_information/ferb_api_health_check_spec.rb [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+          allow(Faraday).to receive(:get).with(ferbapi_url).and_return(response)
+          allow(response).to receive(:status).and_return(468)
+        end
+
+        it 'returns a healthy status' do
+          expect(ferb_api_health_check.check.healthy).to eq false
+        end
+
+        it 'sets an error message' do
+          expect(ferb_api_health_check.check.message).to eq 'ferb api returned 465'
```
[the remark this answers is not in the document]
Comment (AVandana-OSI) [reply spec/system_information/ferb_api_health_check_spec.rb]: Fixing it, will push the changes

File: spec/system_information/ferb_api_health_check_spec.rb [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+            .with(ferbapi_url)
+            .and_throw(Faraday::ConnectionFailed.new('fail'))
+        end
+
+        it 'returns a healthy status' do
+          expect(ferb_api_health_check.check.healthy).to eq false
+        end
+
+        it 'sets an error message' do
+          expect(ferb_api_health_check.check.message) .to eq(error_message)
```
[the remark this answers is not in the document]
Comment (AVandana-OSI) [reply spec/system_information/ferb_api_health_check_spec.rb]: Pushed the changes, Ran the below 3 commands 
bundle
bundle exec rspec
bundle exec rubocop 

All the test pass

File: README.md [hunk 3]
```diff
@@ -21,8 +21,9 @@ bundle
 
 ## Checks
 
-* `:redis` - checks if Redis is available with a ping
-* `:perry` - checks if Perry is available according to its health check
+* `:redis`   - checks if Redis is available with a ping
+* `:perry`   - checks if Perry is available according to its health check
+* `:ferbapi` - checks if ferb api is avilable according to its healt check
```
Comment (edgibbs) [inline README.md]: typo's should be 
`checks if Ferb API is available according to its health check`

File: lib/system_information.rb [hunk 4]
```diff
@@ -10,6 +10,7 @@
 require 'system_information/redis_health_check'
 require 'system_information/system_information_middleware'
 require 'system_information/version'
+require 'system_information/ferb_api_health_check'
```
Comment (edgibbs) [inline lib/system_information.rb]: Small item, but this should be in alpha order.

**Pull request merged**